examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
```

Comment:
Do you need this fixed at 16? The thing is that for FT we need to keep this free, otherwise we cannot estimate the overhead..